### PR TITLE
@alloy - Adds followed_artist module

### LINKED ISSUE #490: [Homepage] Add missing artworks rail for artist on which basis other artist is suggested
As noted by @briansw [here](https://github.com/artsy/force/issues/79#issuecomment-265288454), we currently only show the rail with artworks for the suggested artist, but are omitting the artwork rail for the artist that the suggestion is based on.

### PR DESCRIPTION
**Pull request by broskoski**
Closes #490 

This adds the followed artist module to the list of modules returned (as long as the user is going to receive the `related_artist` module). 

Don't merge until we we are sure it doesn't break the iOS app first (just tested Force, it needs some minimal updates which I will PR). 

screenshot 2016-12-22 09 18 30

### CODE REVIEW

File: test/schema/home/home_page_artwork_modules.js [hunk 1]
```diff
@@ -109,6 +116,7 @@ describe('HomePageArtworkModules', () => {
           'followed_galleries',
           'saved_works',
           'current_fairs',
+          'followed_artist',
           'related_artists',
```
Comment (alloy) [inline test/schema/home/home_page_artwork_modules.js]: Awesome, so these two are always going to be next to each other, regardless of preferred order specified by the client?

Comment (broskoski) [inline test/schema/home/home_page_artwork_modules.js]: Good point, this does break that concept of being able to provide an order beforehand.

Comment (alloy) [inline test/schema/home/home_page_artwork_modules.js]: Basically, I see the two as belonging to the same logical group. i.e. I’d want a client to be able to specify in their preferred order `FOLLOWED_AND_RELATED_ARTIST` and get back the 2 rails in that spot. Do you see a simple way to make that happen?

Comment (broskoski) [inline test/schema/home/home_page_artwork_modules.js]: Not sure they are logically connected (see Brian's comments on #490). I'm fine to do either (make a hard group, or allow them to be re-ordered) way. I'd imagine grouping them together would take a bit more work to sync (with both force and emission).

Comment (alloy) [inline test/schema/home/home_page_artwork_modules.js]: Oh, I had not seen that. I’m kinda surprised by that, though.

**Comment by broskoski** [comment]
Regarding whether this works on emission...

It actually (surprisingly) “works” without any modifications but with a caveat: on force we do a little thing where any rails with zero results get removed immediately (yes slightly hacky), on emission an artist rail with no results looks like this https://www.dropbox.com/s/hnew6v4gsfk5d23/Screenshot%202017-01-03%2015.22.42.png?dl=0 . either we do this on emission also, or figure out a gravity endpoint that will return followed artists with for sale works

**Comment by broskoski** [comment]
Alright @alloy this works fine on emission now. There are a few things to be done to make it work perfectly (i.e. it needs a "See all" link, and probably a follow button) but I'd rather deploy this first and then get the emission polishes up after (so we don't have to sync deploys, if you could even do such a thing with an iOS app...).

We will still need to do a synced deploy for Force and Metaphysics though.

File: schema/home/home_page_artwork_modules.js [hunk 2]
```diff
@@ -139,18 +142,34 @@ const HomePageArtworkModules = {
...
+                        params: {
+                          followed_artist_id: artistPairs[1].sim_artist.id,
+                        },
+                      }
+                    );
+
                     return set(
                       modules,
-                      `[${relatedArtistIndex}].params`,
+                      `[${relatedArtistIndex + 1}].params`,
```
Comment (alloy) [inline schema/home/home_page_artwork_modules.js]: This syntax… wow 👀

Comment (alloy) [inline schema/home/home_page_artwork_modules.js]: Is it basically this?

```js
modules[relatedArtistIndex + 1].params = relatedArtistModuleParams
```

**Comment by alloy** [comment]
Aight, I’ll review the Force PR immediately after merging this.

File: schema/home/home_page_artwork_modules.js (hunk 2 above)
Comment (broskoski) [inline schema/home/home_page_artwork_modules.js]: More or less: https://lodash.com/docs/4.17.2#set

Comment (alloy) [inline schema/home/home_page_artwork_modules.js]: Magic ✨

**Pull request merged**